src/profiles/loopingVideo.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Can we put these profiles next to [the default](https://github.com/cloudinary/cloudinary-video-player/blob/edge/src/config/defaults.js) one?